### PR TITLE
Ports server to 514, updates atmos to TauCeti state. 

### PR DESCRIPTION
**Pull request by drexample**
## About The Pull Request

Guys holy hell Eris at 45+ FPS is so smooth I can't fucking handle it 

## Why It's Good For The Game

https://cdn.discordapp.com/attachments/265411548963405824/891110320087629864/2021-09-25_02-52-53.mp4

Atmos lags less
Atmos machinery lags less
Venry verny good fps looks dope

## Changelog
:cl:
refactor: Made server 514 ready and replaced ticklag with FPS in a few places
refactor: Replaced our air subsystem with TauCeti's air.
/:cl:

### CODE REVIEW

File: code/__DEFINES/tick.dm [hunk 1]
```diff
@@ -22,3 +16,17 @@
...
+
+
+/// Percentage of tick to leave for master controller to run
+#define MAPTICK_MC_MIN_RESERVE 70
+#define MAPTICK_LAST_INTERNAL_TICK_USAGE (world.map_cpu)
+/// Tick limit while running normally
+#define TICK_BYOND_RESERVE 2
+#define TICK_LIMIT_RUNNING (max(100 - TICK_BYOND_RESERVE - MAPTICK_LAST_INTERNAL_TICK_USAGE, MAPTICK_MC_MIN_RESERVE))
+/// Tick limit used to resume things in stoplag
+#define TICK_LIMIT_TO_RUN 70
```
Comment (MLGTASTICa) [inline code/__DEFINES/tick.dm:28]: Line 7 also has this defined

Comment (MLGTASTICa) [reply code/__DEFINES/tick.dm:28]: TICK_LIMIT_MC and TICK_LIMIT_MC_INIT_DEFAULT

File: code/game/world.dm [hunk 2]
```diff
@@ -393,3 +387,11 @@ proc/establish_db_connection()
...
 	maxz++
 	SSmobs.MaxZChanged()
+
+/world/proc/change_fps(new_value = 30)
+	if(new_value <= 0)
+		CRASH("change_fps() called with [new_value] new_value.")
+	if(fps == new_value)
+		return //No change required.
+
+	fps = new_value
```
Comment (Iglpigl) [inline code/game/world.dm:397]: No newline.

File: config/example/config.txt [hunk 3]
```diff
@@ -209,8 +209,8 @@ GITHUBURL https://github.com/Endless-Horizon/CEV-Eris
 
 ## Remove the # to allow special 'Easter-egg' events on special holidays such as seasonal holidays and stuff like 'Talk Like a Pirate Day' :3 YAARRR
 ALLOW_HOLIDAYS
-##Defines the ticklag for the world.  0.9 is the normal one, 0.5 is smoother.
-TICKLAG 0.3
+##Defines the FPSfor the world. 30 is the normal one, 45+ is smoother.
```
Comment (hyperioo) [inline config/example/config.txt:212]:
```suggestion
## Defines the FPS for the world. 30 is the normal one, 45+ is smoother.
```

File: html/add2changelog.html [hunk 4]
```diff
@@ -1,77 +1,77 @@
...
+<option value="sounddel">Removed sound</option>
+</select>
+<button id="addli">+</button>
+</td></tr>
+</table>
+<input type="submit" value="Get .yml code" title="Click here to get result">
+</form>
+<textarea id="result" rows="20" cols="50"></textarea>
+</body>
 </html>
```
Comment (hyperioo) [inline html/add2changelog.html:77]:
```suggestion
</html>

```

File: html/templates/footer.html [hunk 5]
```diff
@@ -1,13 +1,13 @@
...
-<p class="lic"><a name="license" href="http://creativecommons.org/licenses/by-nc-sa/3.0/">Creative Commons License</a><br><i>Except where otherwise noted, CEV Eris is licensed under a <a href="http://creativecommons.org/licenses/by-nc-sa/3.0/">Creative Commons Attribution-Noncommercial-Share Alike 3.0 License</a>.<br></i></p>
-<p class="lic"><i>Some icons by <a href="http://p.yusukekamiyamane.com/">Yusuke Kamiyamane</a>. All rights reserved. Licensed under a <a href="http://creativecommons.org/licenses/by/3.0/">Creative Commons Attribution 3.0 License</a>.</i></p>
-</td></tr></table>
-</body>
-</html>
+<br>
+<b>CEV Eris Development Team</b>
+	<div class = "top">
+	<b>Coders:</b> Clockrigger, Nanako, LethalGhost, Matt, joy ok_hand, fluorescent, ITCups, Camper2012, silentenemy, EvenInDeathIStillServe, Boredone, JTB, Garen7, Kurgis, Nikov, Gray, hitsedesen, Luduk, Macleod962, TorinoFermic, nichlas0010, Cakey, Carbonhell, ACCount, Kmc2000, Granodd, BASIC-BIT, Shazbot, EkuDza<br>
+	<b>Spriters:</b> WatlerJe, Bloodpriest, Shazbot, Toriate, Venligen, Granodd, Jeff, Keekenox<br>
```
Comment (hyperioo) [inline html/templates/footer.html:5]: These are not really up to date

File: html/templates/header.html [hunk 6]
```diff
@@ -1,48 +1,48 @@
...
+</table>
+
+<br><font size='3'><b> CEV Eris Credit List</b></font>
+<table align='center' class="top">
+	<tr>
+		<td valign='top'>
+      <font size='2'><b>Project Head Developer:</b> ACCount<br></font>
+      <font size='2'><b>Project Maintainers:</b> Clockrigger, ACCount, LethalGhost, Matt, joy ok_hand, Little Lost Nestor<br></font>
+			<font size='2'><b>Code:</b> Clockrigger, Nanako, LethalGhost, Matt, joy ok_hand, fluorescent, ITCups, Camper2012, silentenemy, EvenInDeathIStillServe, Boredone, JTB, Garen7, Kurgis, Nikov, Gray, hitsedesen, Luduk, Macleod962, TorinoFermic, nichlas0010, Cakey, Carbonhell, ACCount, Kmc2000, Granodd, BASIC-BIT, Shazbot, EkuDza<br></font>
+			<font size='2'><b>Sprites:</b> WatlerJe, Bloodpriest, Shazbot, Toriate, Venligen, Granodd, Jeff, Keekenox<br></font>
```
Comment (hyperioo) [inline html/templates/header.html:41]: Same here, not up to date

File: html/add2changelog.html (hunk 4 above)
Comment (drexample) [reply html/add2changelog.html:77]: I have no clue what keeps breaking those on my side

**Comment by drexample** [comment]
Whitespace changes exist to clog PRs, god why

**Review by hyperioo — APPROVED** [review]
Seems good, if it has been tested in local and does not blow up the server.

**Review by TheShown911 — APPROVED** [review]
Looks like it worked well on the test server. Don't merge without head dev approval, may need some work with the host.

**Pull request merged**